src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,368 @@
+              <HStack alignItems="center" justifyContent="left">
+                <Box justifyContent={'center'}>
+                  <Checkbox
+                    name="termsAndConditionsAccepted"
+                    defaultChecked={formData['termsAndConditionsAccepted']}
+                    value={termsAndConditionsAccepted}
+                    onChange={e => {
+                      const checked = e.target.checked;
+                      const syntheticEvent = {
+                        target: {
+                          name: e.target.name,
+                          value: checked,
+                        },
+                      };
+                      handleChange(syntheticEvent);
+                      changeTermsAndConditionsAccepted(checked);
+                    }}
+                  ></Checkbox>
+                </Box>
+                <Box marginBottom="0.6vh">
+                  <Text fontSize="xs">
+                    I accept the{' '}
+                    {
+                      <Link
+                        textDecoration="underline"
+                        onClick={() => {
+                          changeTermsAndConditionsIsOpened(true);
+                        }}
+                      >
+                        Terms and Conditions.
+                      </Link>
+                    }
+                  </Text>
+                </Box>
+              </HStack>
+            </Box>
+            <HStack paddingLeft="10vh" paddingRight="10vh" justifyContent="space-between">
+              <Box>
+                <Button
+                  variant="outline"
+                  borderColor="#319795"
+                  color="#319795"
+                  width="9vh"
+                  type="submit"
+                  onClick={prevStep}
+                >
+                  Back
+                </Button>
+              </Box>
+              <Box>
+                <Button
+                  type="submit"
+                  background="#319795"
+                  width="9vh"
+                  onClick={async e => {
+                    if (!termsAndConditionsAccepted) {
+                      changeSubmitAndAcceptedFalse(true);
+                    } else {
+                      await handleSubmit(e);
+                    }
+                  }}
+                  colorScheme="blue"
+                >
+                  Submit
+                </Button>
+              </Box>
+            </HStack>
+            <>
+              <Box paddingLeft="10vh" paddingRight="10vh">
+                {submitAndAcceptedFalse ? (
+                  <>
+                    <Alert borderRadius="2vh" status="error">
+                      <Flex align="center" justify="space-between" width="100%">
+                        <Box>
+                          <HStack>
+                            <AlertIcon />
+                            <AlertTitle>You haven't accepted the Terms and Conditions</AlertTitle>
```

Comment:
🚫 **[eslint]** <[react/no-unescaped-entities](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unescaped-entities.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.